app/constants.py

```diff
@@ -23,7 +23,7 @@ def __init__(self):
         self.current_semester_id = self.semester.get(self.current_semester, '2020-秋')
 
     def _load_config_yaml(self) -> schemas.YamlConfig:
-        with open(os.path.join(self.app_path, 'config.yaml')) as fp:
+        with open(os.path.join(self.app_path, 'config.yaml'),encoding='utf-8') as fp:
```

Comment:
It would be better to place a space after the symbol of comma, you can find more conventions of cleaning code on [here](https://liaoguoyin.com/tags/clean), or you can use some plugins for linting before pushing to GitHub.